src/audio-service.ts

```diff
@@ -0,0 +1,27 @@
+import { writable } from 'svelte/store';
+
+const _audio = new Audio();
```

Comment:
issue: I'm getting an exception here:
```
ReferenceError: Audio is not defined
    at /home/illright/work/cast/src/audio-service.ts:3:19
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async instantiateModule (/home/illright/work/cast/node_modules/.pnpm/vite@2.7.13/node_modules/vite/dist/node/chunks/dep-f5552faa.js:60105:9)
```

Reply:
Should work now as I pass an `HTMLAudioElement` bound from the layout.